Cubical/CW/Subcomplex.agda

```diff
@@ -133,6 +133,46 @@ realiseSubComplex n C =
   compIso (equivToIso (complex≃subcomplex C n flast))
           (realiseFin n (finSubComplex C n))
 
+niceFinCWskel : ∀ {ℓ} (n : ℕ) → finCWskel ℓ n → finCWskel ℓ n
```

Comment:
Please improve the comments in this file, so it is easier to understand what this is doing.